README.md

```diff
@@ -1,6 +1,10 @@
-Welcome to the Cinder for Android and Linux ports! Please see **[Cinder for Android](https://github.com/cinder/Cinder/wiki/Cinder-for-Android)** and **[Cinder for Linux](https://github.com/cinder/Cinder/wiki/Cinder-for-Linux)** for details on how to get going.
+<p align="center">
+  <img src="https://libcinder.org/static/images/notes/vulkan/Vulkan_Mar15.svg" alt="Vulkan Logo" width="256" height="auto"/>
+</p>
+
+Welcome to the experimental Vulkan implementation in Cinder.
 
-If you run into any issues or have any questions, please [visit the forums](https://forum.libcinder.org).
```

Comment:
Should this be reverted to point to libcinder.org? I assume we'll have a guide for vulkan at some point, to cover that